juzzyPython/type1/system/T1_Antecedent.py

```diff
@@ -87,8 +87,8 @@ def getMax(self,tNorm: int) -> float:
             valxmax = 0
             domain = self.input.getDomain().getRight() - self.input.getDomain().getLeft()
             incr = domain/(domain*50)
-            x = 0
-            for i in range((domain*50)+1):
+            x = self.input.getDomain().getLeft()
+            for i in range((int(domain)*50)+1):
```

Comment:
Thank you for your contribution :) The change LGTM, can I also please ask that you add these changes to the IT2_Antecedent to this PR, as well as an image of SimpleIT2FLS_test.py and SimpleIT1FLS_test.py unit tests passing. Thanks! Please ask if you have any questions.